js/__internal/grids/grid_core/data_controller/module.ts

```diff
@@ -542,7 +542,8 @@ export class DataController extends ControllerWithDataMixin {
     this.pushed.fire(changes);
   }
 
-  fireError() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  fireError(...args: any[]) {
```

Comment:
If possible let's replace ```arguments``` with ```args``` inside this method.

If it is not possible, let's remove the ```...ags``` from the method signature.

Anyway it's better don't mix up the ```args``` and ```arguments```